tests/unit/test_utils.py

```
@@ -334,3 +337,237 @@ def test_get_default_headers_immutable(self):
+
+    def setup_method(self):
+        utils_module.configs.json_output = False
+
+    PATIENT_WITH_META = {
+        "resourceType": "Patient",
+        "id": "p1",
+        "gender": "male",
+        "meta": {"versionId": "1", "lastUpdated": "2024-01-01"},
+        "text": {"status": "generated", "div": "<div>...</div>"},
+        "name": [{"family": "Smith"}],
+    }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)